Utilities/Pipelines/Tasks/xdk-build.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  - name: platform
+    type: string
+  - name: configuration
+    type: string
+
+steps:
+  - template: checkout.yml
+  
+  - task: PkgESSetupBuild@12
+    displayName: PkgES build setup
+    inputs:
+      useDfs: false
+      productName: XboxLiveSDK
+      branchVersion: true
+      extSync: true
+      disableMsbuildVersion: true
+
+  - task: BatchScript@1
+    displayName: fetchTools.cmd
+    inputs:
+      filename: Utilities/Pipelines/Scripts/fetchTools.cmd
+      arguments: '"$(PAT)" "$(BuildTools)"'
+      modifyEnvironment: false
+
+  - task: BatchScript@1
+    displayName: vsvars.cmd
+    inputs:
+      filename: Utilities/Pipelines/Scripts/vsvars.cmd
+      modifyEnvironment: true
+
+  - task: BatchScript@1
+    displayName: DurangoVars
+    inputs:
+      filename: '"C:\Program Files (x86)\Microsoft Durango XDK\xdk\DurangoVars.cmd"'
+      arguments: VS2017
+      modifyEnvironment: true
+
+  - task: BatchScript@1
+    displayName: xdkvars.cmd
+    inputs:
+      filename: Utilities/Pipelines/Scripts/xdkvars.cmd
+      modifyEnvironment: true
+
+  - task: VSBuild@1
+    displayName: libHttpClient.vs2017.sln Durango
+    inputs:
+      solution: libHttpClient.vs2017.sln
```

Comment:
Do we need a vs2019 build for this as well, or does it only live in vs2017?

Reply:
No VS2019 support in this one